e2e-tests/page-objects/shared/profile-page.js

```diff
@@ -0,0 +1,44 @@
+/*!
+ * Copyright (c) 2015-2016, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+'use strict';
+
+const util = require('./util');
+
+class ProfilePage {
+
+  constructor() {
+    this.$$claims = $$('td');
```

Comment:
This seems like it could be fragile, as it is only looking for a `td` element.

@robertjd It might make sense to update our frontend to include an `id` attribute with the `claim` name.
Ex:
```
<tr>
  <td id="td-{{claim.name}}">
    {{claim.name}}
  </td>
  <td id="td-{{claim.value}}">
    {{claim.value}}
  </td>
</tr>
```

Reply:
Done. Looking for 'claim-email' element id